src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  let cost = 0;
+  const discountThree = 20;
+  const discountweek = 50;
+
+  if (days < 3) {
+    cost = days * 40;
+  }
+
+  if (days >= 7) {
+    cost = (days * 40) - discountweek;
+  }
 
+  if (days >= 3 && days < 7) {
```

Comment:
Try to avoid magic numbers. All numbers that are unclear from the context should be put into constants with a name that is understandable in terms of content